File: DataArt.TaskManager.Entities/Task.cs
namespace DataArt.TaskManager.Entities
{
    public class Task
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public Category Category { get; set; }

        public bool IsDone { get; set; }

        public TaskState State { get; set; }
    }
}

File: DataArt.TaskManager.Web/Scripts/spa/Task.js
function Task(id, isDone, category, description, state) {
    var self = this;

    self.id = id;
    self.isDone = isDone;
    self.category = ko.observable(category);
    self.title = ko.observable(description);
    self.state = ko.observable(state || 0);
}